### DISCUSSION TITLE
[question] Aeotec Ltd. / Range Extender 7 and Razberry 7 Pro

### DISCUSSION THREAD

**Discussion by posixx** [zwave-js/zwave-js-ui#3865] [Q&A]
Hello,

I have setup a new network, coming from zwave-js with aeotec z-stick gen5. Everything is working using Razberry 7 Pro, only thing i notice is it looks like the Range Extender 7 isn't extending my range. I already tried healing the network but is still displayed as "standalone" device:

![afbeelding](https://github.com/user-attachments/assets/d722c1b1-ddaa-41c3-8e9c-bf720b2ee3ce)

Does anyone know if this configuation is supported? It looks like my network can function without the extender but before i remove it i want to make sure how to determine if the extender is doing something or not..

To be complete; i didn't configure a value for power levels (normal and measured) maybe i can finetune that so the extender isn't necessary? 

![afbeelding](https://github.com/user-attachments/assets/7570d4c3-b214-40ee-8274-b7b235339375)

Thanks for you help in avance to help me understand zwave better!

**Comment by robertsLando** [comment]
In the way your network is routing right now the range extender isn't doing anything.Your devices should use the range extender as a repeter, you can force this by changing the priority route of such devices to make them using the extender as a repeter before the controller.

cc @AlCalzone
